packages/docs/base/color.md

```diff
@@ -1,57 +1,614 @@
 # Color
 
-## Neutral palette
-
-<ul class="sample--color-list">
-  <li class="sample--color is-sample-gray-000"></li>
-  <li class="sample--color is-sample-gray-100"></li>
-  <li class="sample--color is-sample-gray-200"></li>
-  <li class="sample--color is-sample-gray-300"></li>
-  <li class="sample--color is-sample-gray-400"></li>
-  <li class="sample--color is-sample-gray-600"></li>
-  <li class="sample--color is-sample-gray-900"></li>
-</ul>
-
-## Primary palette
-
-### Blue
-<ul class="sample--color-list">
-  <li class="sample--color is-sample-blue-000"></li>
-  <li class="sample--color is-sample-blue-300"></li>
-  <li class="sample--color is-sample-blue-500"></li>
-  <li class="sample--color is-sample-blue-900"></li>
-</ul>
-
-### Green
-<ul class="sample--color-list">
-  <li class="sample--color is-sample-green-000"></li>
-  <li class="sample--color is-sample-green-300"></li>
-  <li class="sample--color is-sample-green-500"></li>
-  <li class="sample--color is-sample-green-900"></li>
-</ul>
-
-### Red
-<ul class="sample--color-list">
-  <li class="sample--color is-sample-red-000"></li>
-  <li class="sample--color is-sample-red-300"></li>
-  <li class="sample--color is-sample-red-500"></li>
-  <li class="sample--color is-sample-red-900"></li>
-</ul>
-
-### Yellow
-<ul class="sample--color-list">
-  <li class="sample--color
-   is-sample-yellow-000"></li>
-  <li class="sample--color is-sample-yellow-300"></li>
-  <li class="sample--color is-sample-yellow-500"></li>
-  <li class="sample--color is-sample-yellow-900"></li>
-</ul>
-
-
-## Secondary palette
-
-<ul class="sample--color-list">
-  <li class="sample--color is-sample-purple-500"></li>
-  <li class="sample--color is-sample-turquoise-500"></li>
-  <li class="sample--color is-sample-orange-500"></li>
-</ul>
+Here's a quick overview of the colors available in Odyssey and some guidelines for their use.
+
+<figure class="ods-table--figure">
+  <figcaption class="ods-table--figcaption">
+    Neutral Palette
+  </figcaption>
+  <table class="ods-table">
+    <thead>
+      <tr>
+        <th scope="column">Swatch</th>
+        <th scope="column">Name</th>
+        <th scope="column">Hex</th>
+        <th scope="column">✅ Do</th>
```

Comment:
@joejezowski-okta @arnoldsandoval-okta @johnintrater-okta 

As a general note, the Do's and Don'ts here reflect the current use in Odyssey but don't give much general guidance:

> Button 500: Links, buttons, icons, hover states, selected states

VS something like

> Blue 500: Best used for indicating interactive UI

Any thoughts there?

Reply:
Discussed offline - we'll revisit these as we close in on a Token-based approach.